building-a-preview/automate-previews/index.md

```diff
@@ -0,0 +1,189 @@
+# Automate Previews
+
+We love automatically generating Previews from new pull requests - we think it's
+one of Tugboat's best features! If you're using a
+[git provider integration](../../setting-up-tugboat/index.md#connect-with-your-provider),
+you'll be able to automate Preview actions in a few different ways:
+
+- [Auto-generate Previews](#auto-generate-previews)
+- [Auto-update Previews](#auto-update-previews)
+- [Auto-delete Previews](#auto-delete-previews)
+
+## Auto-generate Previews
+
+Tugboat provides two ways to automatically generate Previews when pull requests
```

Comment:
I think this section needs a brief revisit. We took out one of the two ways that were originally mentioned here.